demos/sslecho/main.c

```diff
@@ -331,8 +335,10 @@ int main(int argc, char **argv)
     if (server_skt != -1)
         close(server_skt);
 
-    OPENSSL_free(txbuf);
-    OPENSSL_free(rxbuf);
+    if ( txbuf && (txcap>0) )
```

Comment:
Please remove the extra parentheses. There should be no space after `(` and before `)`, there should be spaces around `>` and the condition for txbuf should be `txbuf != NULL`.

Also please remove the `{` and `}` as  there is only a single statement in the if and indent the `free()` properly (should be indented by 8 spaces instead of a tab)